go-controller/pkg/ovn/endpoints.go

```diff
@@ -282,3 +252,20 @@ func (ovn *Controller) clearVIPsAddRejectACL(svc *kapi.Service, lb, ip string, p
 		klog.Errorf("Error in clearing endpoints for lb %s: %v", lb, err)
 	}
 }
+
+// hasHostEndpoints determines if a slice of endpoints contains a host networked pod
```

Comment:
This function is duplicated, it's defined in `pkg/ovn/controller/services/utils.go ` as well. I think you can just use that one and remove this?

Reply:
a lot of the functions from ovn package are duplicated into controller/services. This is due to cyclical dependency. We are going to eventually remove all of the ones from ovn pkg. This is legacy only.